### PR TITLE
Adds the filter:uploadStored hook which fires after the file is saved…

### PR DESCRIPTION
**Pull request by dbolack-ab**
… in the async waterfall and passes:

                       var storedFile = {
                       url: nconf.get('relative_path') + upload.url,
                       path: upload.path,
                       name: uploadedFile.name,
               }
               plugins.fireHook( 'filter:uploadStored', { uploadedFile: uploadedFile, storedFile: storedFile } );

### CODE REVIEW

File: src/controllers/uploads.js [hunk 1]
```diff
@@ -227,11 +227,13 @@ function saveFileToLocal(uploadedFile, callback) {
...
-				url: nconf.get('relative_path') + upload.url,
-				path: upload.path,
-				name: uploadedFile.name,
-			});
+			var storedFile = {
+			url: nconf.get('relative_path') + upload.url,
+			path: upload.path,
+			name: uploadedFile.name,
+		}
+		plugins.fireHook( 'filter:uploadStored', { uploadedFile: uploadedFile, storedFile: storedFile } );
```
Comment (pitaj) [inline src/controllers/uploads.js]: Add `next` as the last argument to the `fireHook` function.

Also, your formatting doesn't match our styleguide which is causing tests to pass.

File: src/controllers/uploads.js [hunk 2]
```diff
@@ -227,11 +227,14 @@ function saveFileToLocal(uploadedFile, callback) {
...
-			next(null, {
+			var storedFile = {
 				url: nconf.get('relative_path') + upload.url,
 				path: upload.path,
 				name: uploadedFile.name,
-			});
+			};
+
+			plugins.fireHook('filter:uploadStored', { uploadedFile: uploadedFile, storedFile: storedFile }, next);
+			next(null, storedFile);
```
Comment (pitaj) [inline src/controllers/uploads.js]: `plugins.fireHook` calls next itself when it is done, so don't call next a second time.

File: src/controllers/uploads.js [hunk 3]
```diff
@@ -227,11 +227,13 @@ function saveFileToLocal(uploadedFile, callback) {
...
 		function (upload, next) {
-			next(null, {
+			var storedFile = {
 				url: nconf.get('relative_path') + upload.url,
 				path: upload.path,
 				name: uploadedFile.name,
-			});
+			};
+
+			plugins.fireHook('filter:uploadStored', { uploadedFile: uploadedFile, storedFile: storedFile }, function () { next(null, storedFile); });
```
Comment (pitaj) [inline src/controllers/uploads.js]: `function () { next(null, storedFile); }` =>

```
function (err, data) {
  if (err) {
    return next(err);
  }

  next(null, data.storedFile);
}
```

**Comment by dbolack-ab** [comment]
This green?

**Pull request merged**